components/common/contracts-shortcuts.ts

```diff
@@ -30,6 +32,8 @@ export async function accountBalances(account: EthAccount, contracts: Contracts)
     erc1155BalanceOf(account.address, contracts.debtCouponToken),
     erc1155BalanceOf(account.address, (contracts.bondingToken as unknown) as ERC1155Ubiquity),
     contracts.usdc.balanceOf(account.address),
+    erc1155BalanceOf(account.address, (contracts.uBOND as unknown) as ERC1155Ubiquity), // uBOND
```

Comment:
uBOND will be like uAR, it's an ERC20 token, so a simple balanceOf should suffice.